packages/apps/fortune/exchange-oracle/server/src/modules/webhook/webhook.dto.ts

```diff
@@ -53,6 +59,7 @@ export class WebhookDto {
   @ApiProperty({
     name: 'event_data',
   })
```

Comment:
```suggestion
  @ApiProperty({
    name: 'event_data',
    required: false,
  })
```

Without this swagger marks it as required